src/main/resources/hy.yml

```diff
@@ -162,7 +162,7 @@ hy:
         NU: Նիուե
         NF: Նորֆոլկ Կղզի
         MP: Հյուսիսային Մարիանյան Կղզիներ
-        NO: Նորվեգիա
+        'NO': Նորվեգիա
```

Comment:
Why is this necessary?

(I don't disagree, just wondering if it's something I should watch for)

Reply:
Is NO a reserved word?

Reply:
@kingthorin I am glad you asked it :)
The problem is that Yaml parser reads `NO: Նորվեգիա` as `false: Նորվեգիա` :) :) :) Hilarious!

By the way, is `country_by_code` property in yml files needed at all?
Afaik it's not used anywhere...